### ISSUE TITLE
Allow array of strings to be passed to `entrypoint`

### ISSUE THREAD

**Issue by peacememories** [hercules-ci/arion#215]
The docker-compose (and docker) spec allows (and even prefers, from what I can tell) users to supply the `entrypoint` as a list of command and arguments. It would be nice to have this ability in arion as well.

**Comment by roberth** [comment]
Absolutely. I would expect this to be as simple as changing a `str` to `either str (listOf str)`, and change a test to use it that way. I see little use in testing the old way.
